Sources/VaporToolbox/Build.swift

```diff
@@ -25,107 +30,100 @@ public final class Build: Command {
     }
 
     public func run(arguments: [String]) throws {
-        if arguments.options["clean"]?.bool == true {
-            let clean = Clean(console: console)
-            try clean.run(arguments: arguments)
-        }
+        try clean(arguments)
+        try fetch(arguments)
+        try build(arguments)
+        try run(arguments)
+    }
 
-        if arguments.options["fetch"]?.bool != false {
-            let fetch = Fetch(console: console)
-            try fetch.run(arguments: [])
-        }
+    private func clean(_ arguments: [String]) throws {
+        guard arguments.flag("clean") else { return }
+        let arguments = arguments.removeFlags(["clean"])
+        let clean = Clean(console: console)
+        try clean.run(arguments: arguments)
+    }
 
-        if arguments.options["modulemap"]?.bool != false {
-            do {
-                let mod = try console.backgroundExecute(program: "/bin/sh", arguments: ["-c", "ls Packages | grep CLibreSSL"]).trim()
-                _ = try console.backgroundExecute(program: "ls", arguments: ["Packages/\(mod)/Sources/CLibreSSL/include/module.modulemap"])
-            } catch {
-                // not found
-                do {
-                    _ = try console.backgroundExecute(program: "/bin/sh", arguments: ["-c", "printf \"module CLibreSSL {\\n    header \\\"CLibreSSL.h\\\"\\n    link \\\"CLibreSSL\\\"\\n}\" > Packages/CLibreSSL-1.0.0/Sources/CLibreSSL/include/module.modulemap"])
-                } catch {
-                    console.warning("Could not add CLibreSSL Module Map: \(error)")
-                }
-            }
-        }
+    private func fetch(_ arguments: [String]) throws {
+        // unless explicitly false, treat as true
+        let shouldFetch = arguments.option("fetch")?.bool ?? true
```

Comment:
It's going to happen anyways, and looks nice in readout so user can follow steps. Worth keeping I think 👍